### ISSUE TITLE
knitting error

### ISSUE THREAD

**Issue by knboysen** [ucsb-bren/students#108]
My tidyr code seems to work fine, but the kniting process fails when I try to use "gather"  code. The error reads: 

"Error in eval(expr, envir, enclos): not compatible with STRSXP calls: <Anonymous>...<Anonymous> -> select_Vars_ -> combine_vars ->.Call"

any hints? I submitted a pull request so you can look at my code (kboysen.rmd)

**Comment by bbest** [comment]
Hi @knboysen,

The only problem was a trailing `%>%` which was incompatible with the following `head(co2_long)`.

Original failing:

``` r
co2_long<- co2 %>% 
  gather("Year", "CO2", 2:46) %>%
  head(co2_long)
```

Corrected:

``` r
co2_long<- co2 %>% 
  gather("Year", "CO2", 2:46)
head(co2_long)
```

If you want to get really fancy, you can use the extra "tee" operator `%T>%` in magrittr to not only return the full data frame to co2_long, but also tee off the outputting of the head.

``` r
library(magrittr) 

co2_long<- co2 %>% 
  gather("Year", "CO2", 2:46) %T>%
  head()
```

**Comment by bbest** [comment]
The other issue giving you this actual error was leaving the `eval=F` in the "read co2" chunk. So `co2` did not exist when gather tried to operate on it. Wish the error message was more helpful however, instead of this:

```
Error in eval(expr, envir, enclos): not compatible with STRSXP calls: ... -> select_Vars_ -> combine_vars ->.Call
```

After class Friday, I added a note about this in [wk04_tidyr.Rmd](https://github.com/ucsb-bren/env-info/blob/e98c4e755ce2bb137085bd526c3f4c09c75418a6/wk04_tidyr.Rmd#assignment-individual):
- You will want to change the R chunk option from evaluate = FALSE `{r, eval=F}` to evaluate = TRUE `{r, eval=T}`. This is especially the case for `read co2` R chunk which reads in the `co2` data frame, since you will create additional R chunks which expect this object to already exist.

But then when revisiting the materials for the 7 students who missed Friday due to the fisheries modeling course, I  decided to simplify the instructions by removing the `eval=F` from the individual chunks in the current [wk04_tidyr.Rmd](https://github.com/ucsb-bren/env-info/blob/gh-pages/wk04_tidyr.Rmd) and removed that note so when students copy chunks they default to `eval=T`. For rendering the instructions, I made the default to `eval=F` by adding this hidden chunk (`echo=F`) at the top:

``` r setup, include=FALSE
knitr::opts_chunk$set(echo = TRUE, eval=FALSE)
```

TMI, I know :blush:
